gma.lua

```diff
@@ -53,28 +53,28 @@ end
 function gma.git_version()
 end
 
----Export a table as a file???
+---Export a table as a file. By default, MA uses XML as a file type.  Check @{gma.export_csv()} and @{gma.export_json()} for other options.
 --
---@tparam string filename Filename to export to???
---@tparam table export_data Table object to export???
+--@tparam string filename Filename to export to. Exports to \MA Lighting Technologies\grandma\gma2_V_<YourVersion>\temp
```

Comment:
Is this tested on a console, onPC, or both?

Reply:
Forgot to add, that it is tested only on onPC version.